3d_segmentation/torch/unet_evaluation_array.py

```diff
@@ -73,10 +73,14 @@ def main(tempdir):
             roi_size = (96, 96, 96)
             sw_batch_size = 4
             val_outputs = sliding_window_inference(val_images, roi_size, sw_batch_size, model)
-            val_outputs = post_trans(val_outputs)
+            val_outputs = torch.stack([post_trans(i) for i in decollate_batch(val_outputs)])
```

Comment:
no need `torch.stack()`.

Reply:
Here `NiftiSaver` is used, and it needs tensor/numpy array input

Reply:
`SaveImage` transform supports NIFTI format, could you please change to use `SaveImage` instead?